oauth2/docs/README.md

```diff
@@ -0,0 +1,124 @@
+Tidepool's OAuth 2.0
+=========
+
+Apps connect to Tidepool using OAuth 2.0, the standard used by most APIs for authenticating and authorizing users.
+
+# Initial Setup
+
+Before you can start using OAuth2 with your application, you’ll need to tell Tidepool a bit of information about your application
+
+## Register your application here.
+
+``http://localhost:8009/oauth/signup``
+
+Tell us about the app
+* Set your application name
+* Set your redirect uri
+
+Create a platform user
+* email
+* password
+
+```
+curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -d 'usr_name=Docs+Example&uri=http%3A%2F%2Flocalhost%3A14000&email=docs%40tidepool.org&password=docs4life&password_confirm=docs4life' http://localhost:8009/auth/oauth2/signup
+```
+
+Make a note of both your client_id and client_secret.
+
+```
+client_id=d995aaabe7
+client_secret=dd85baba6e0816662e4b16d32242f6526cf3e35a
+```
+
+### Notes:
+
+* What is a platfrom user?
+ * This is your applications account on the Tidepool platfrom
+
+* What is the redirect URI?
+ * The redirect URI is the URL within your application that will receive the OAuth2 credentials.
+
+* Scopes available:
+  * Requests uploading of data on behalf
```

Comment:
Clearer way to say this? Feels like it just cuts off... Requests uploading on behalf of who?